web/templates/layout/app.html.eex

```diff
@@ -24,6 +24,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= title_or_default(assigns[:title]) %></title>
+    <%= if description = assigns[:description] do %>
+    <meta name="description" content="<%= description %>">
```

Comment:
Cool -- I've worked on projects that don't indent conditionals because they didn't want the HTML output to be indented, but I will follow your style here.